src/uucore/src/lib/features/format/num_format.rs

```diff
@@ -326,6 +315,18 @@ impl Formatter for Float {
     }
 }
 
+fn get_flag<T: PartialOrd + Default>(sign: PositiveSign, x: &T) -> String {
```

Comment:
get_flag is a bit too generic 
maybe call it get_sign_indicator ?

Reply:
Indeed. Changed it now. thanks for the advice.